.github/workflows/pimcore-demo.yml

```
@@ -54,16 +62,22 @@ jobs:
               sample-project
 
     - name: Smoke-test compose file
+      env:
+          DOCKER_DATABASE: "${{ matrix.database }}"
       run: |
         # Echo commands and terminate on first error
         set -ex
         
         # Check (lint) the compose file
         docker-compose -v
         cd sample-project/
-        docker-compose config
+        docker-compose -f docker-compose.yaml -f .github/ci/files/docker-compose.yaml config
 
     - name: Test pimcore installation
+      continue-on-error: true
```

we still experience some false-negative test failing during db creation, better to keep it always true to not stopping all the others

---

```suggestion
      continue-on-error: ${{ matrix.experimental }}
```
why not use experimental flag from the matrix?